clients/cloud/python/README.md

```diff
@@ -17,6 +17,24 @@ sasl.username=<api-key-id>
 sasl.password=<secret-access-key>
 ```
 
+## Configure SSL trust store
+
+Depending on your operating system or Linux distro you may need to take extra steps to set up the SSL CA root certificates.
+
```

Comment:
It would be helpful to show an example error message, e.g. "If your systems does not have the SSL CA root certificates properly set up, here is an example of an error message you may see."

Reply:
Fixed in https://github.com/confluentinc/examples/pull/144/commits/8f0d160ab9f7b19688b5676c1ace4a59885c8b73

Reply:
👍